docs/events.md

```diff
@@ -153,6 +155,18 @@
 
 #
 
+> ### <a name="USER_SHOWED_CAPTIONS"></a>USER_SHOWED_CAPTIONS
+>
+> Fires when the user showed the captions by the CC button.
+
+#
+
+> ### <a name="USER_HID_CAPTIONS"></a>USER_HID_CAPTIONS
+>
+> Fires when the user hid the captions by the CC button.
```

Comment:
Hid is a typo, did you mean hide?

Reply:
Hid = Hide in past (and showed for show)